clang/tools/pattern-rewriter/src/driver.cpp

```diff
@@ -16,6 +16,16 @@
 #include "llvm/Support/CommandLine.h"
 #include "llvm/Support/YAMLTraits.h"
 
+static constexpr auto HelpMessage = R"--(
+DESCRIPTION:
+
+SYCLomatic provides the pattern-rewrite tool to automate the manual adjustment that is sometimes needed in the code before or after migration. The pattern-rewrite tool uses user-defined patterns and can be applied to any code.
```

Comment:
```suggestion
SYCLomatic provides the tool pattern-rewrite to apply the code update and adjustment automatically based on user-defined pattern(s) in Yaml format, it can process on any code before or after migration. 
For example, automate the manual adjustments after migration to SYCL with defined pattern(s) in YAML format, enabling you to rerun migration multiple times and reapply your adjustments.

```